### ISSUE TITLE
Zoom

### ISSUE THREAD

**Issue by radog** [pchen66/panolens.js#4]
Hi,

 1. How can initialize panorama with a maximum zoom out? Or how to set zoom in panorama?

 2. It is possible draped cursor opposite directions?

**Comment by pchen66** [comment]
Yes, both are possible. `PANOLENS.Viewer` exposes it's camera to you can do any camera effect you would like. The default camera is `[THREE.PerspectiveCamera](http://threejs.org/docs/#Reference/Cameras/PerspectiveCamera)`
(1) To set the zoom level (as in changing the field-of-view) default is 60, min is 20 and max 120. Change camera's fov value and you should see the difference. Don't forget to `updateProjectionMatrix`.
```javascript
viewer.getCamera().fov = 30;
viewer.getCamera().updateProjectionMatrix();
```
(2) If you want opposite dragging direction
```javascript
viewer.getControl().rotateSpeed *= -1;
viewer.getControl().momentumScalingFactor *= -1;
```
Do you think the opposite cursor direction makes more sense?

**Comment by radog** [comment]
Thank you, exactly what I needed!

People are accustomed to the opposite direction of the cursor on Google Street View. :-)

**Comment by pchen66** [comment]
Great. Thanks for sharing! You have pretty nice panoramas. Your questions are in fact good suggestions too. Implemented in #6. The viewer will accept `cameraFov` and `reverseDragging` during the initialization. Like this example
```javascript
viewer = new PANOLENS.Viewer( { cameraFov: 80, reverseDragging: true } );
``` 
so you don't have to manually call 
```javascript
viewer.getCamera().fov = 30;
viewer.getCamera().updateProjectionMatrix();
viewer.getControl().rotateSpeed *= -1;
viewer.getControl().momentumScalingFactor *= -1;
```
If you still want to change these settings on the fly
```javascript
viewer.setCameraFov(100);
viewer.reverseDraggingDirection();
```
